core/App/ehsm_napi.cpp

```diff
@@ -84,34 +83,50 @@ void NAPI_Finalize(){
     Finalize();
 }
 
-/*
-@return
-[string] json string
+/**
+ * @brief Create key and save the parameters when using the key for encrypt, decrypt, sign and verify
+ * 
+ * @param paramJson Pass in the key parameter in the form of JSON string
+ * @param paraJson will carry with keyspec, padding mode, digest mode, purpose and origin
+ * other parameters for create key will also in @param paraJson if need
```

Comment:
you need to list all the parameters that currently support which in the #line 93 format.

same with other APIs.